views/js/pciCreator/ims/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -443,26 +478,33 @@ define([
             /**
              * Gap mode only: fill the mathfield markup with the math expression before creating the MathQuill instance
              * @param {String} latex - the math expression with gaps
+             * @param {String} index
              */
-            setMathStaticContent: function setMathStaticContent(latex) {
+            setMathStaticContent: function setMathStaticContent(latex, index) {
+                const [inputIndex] = this.inputs.keys();
+                if (typeof index === 'undefined') {
+                    index = inputIndex;
+                }
                 latex = latex
                     .replace(/\\taoGap/g, '\\MathQuillMathField{}')
                     .replace(/\\taoBr/g, '\\embed{br}');
-                this.$input.text(latex);
+                $(this.inputs.get(index)).text(latex);
```

Comment:
**suggestion:** Since it is already a jQuery element, it could be done this way:
```suggestion
                this.inputs.eq(index).text(latex);
```

See: https://api.jquery.com/eq/

Reply:
`this.inputs` is a Map object here, the get method here returns the jQuery object